src/Update.hpp

```diff
@@ -0,0 +1,71 @@
+#ifndef CANOPEN_MASTER_UPDATE_HPP
+#define CANOPEN_MASTER_UPDATE_HPP
+
+#include <cstdint>
+
+namespace canopen_master {
+    /**
+     * Class used to represent the state update as returned by
+     * Slave::process
+     */
+    class Update {
+        uint32_t mAckedObjectID;
```

Comment:
Actually, the rest of the (pre-existing) canopen_master package is using `mCamelCase`. I think it would be best to stick with the style the code uses already. Thoughts ?

Reply:
(this would obviously mean renaming `m_can_open` in `Slave`)

Reply:
Sounds good... There's also an `m_slave`

Reply:
Done